chromebook-setup.sh

```diff
@@ -845,7 +849,7 @@ ensure_command realpath realpath
 ensure_command sgdisk gdisk
 ensure_command lz4 lz4
 ensure_command mkfs.ext4 e2fsprogs
-ensure_command mkimage u-boot-tools
+ensure_command mkimage uboot-tools
```

Comment:
Hmm, so debian names this package as u-boot-tools whether fedora names it as uboot-tools. Not sure how e need to handle these cases but in any case I think it should be a separate patch. So I'd prefer do not change it in this PR.

Reply:
okay, i see. fedora names it otherwise. I'd create a separate patch to fix that then